config/initializers/secure_headers.rb

```diff
@@ -62,7 +62,7 @@
 SecureHeaders::Configuration.override(:saml) do |config|
   provider_attributes = SERVICE_PROVIDERS.values
 
-  acs_urls = provider_attributes.map { |hash| hash['acs_url'] }
+  acs_urls = provider_attributes.map { |hash| hash['acs_url'] }.compact
```

Comment:
Yeah I assumed this was for form submissions only

OpenID doesn't do POSTs/form submits to other domains, it only sends a redirect. Also, I chose to use the config key `redirect_url` instead of re-using `acl_url` for OpenID connect. So what that did is create entries with no `acl_url` which is why I added the `compact` (so it doesn't blow up at the next statement)